src/environmentd/src/lib.rs

```diff
@@ -447,12 +453,15 @@ impl Listeners {
             }
         }
 
-        let openable_adapter_storage = catalog_opener(
+        let mut openable_adapter_storage = catalog_opener(
             &config.catalog_config,
             &config.controller,
             &config.environment_id,
         )
         .await?;
+        let enable_persist_txn_tables_stash_ld = openable_adapter_storage
+            .get_enable_persist_txn_tables()
+            .await?;
```

Comment:
Can you add a comment here to indicate that this isn't actually the current value of LD, but what the value in LD was the last time this environment was running?

Reply:
Done